R/1.0_module_teal_modules.R

```diff
@@ -345,3 +356,14 @@ srv_teal_module.teal_module <- function(id,
     USE.NAMES = FALSE
   )
 }
+
+#' Get code that creates a backup of the original (unfiltered) data
+#'
+#' @param datanames (`character`) names of datasets
+#' @return A character vector with code lines that copy existing `<datanames>` to
+#' `<datanames>_raw`.
+#'
+#' @keywords internal
+.get_raw_code <- function(datanames) {
+  paste(sprintf("%s_raw", datanames), "<-", datanames, collapse = "\n")
```

Comment:
```suggestion
  sprintf("%1$s <- %1$s_raw", datanames)
```

Reply:
Looking how simple this function can be I suggest to just make inline `sprintf` instead of making extra function

Reply:
True, I don't use sprintf enough